src/libraries/System.Runtime.InteropServices/tests/System.Runtime.InteropServices.UnitTests/System/Runtime/InteropServices/HandleCollectorTests.cs

```diff
@@ -155,11 +157,13 @@ public static void TestHandleCollector()
 
         private sealed class HandleLimitTester
         {
-            private HandleCollector _collector;
+            private readonly HandleCollector _collector;
+            private readonly int[] _pressure;
 
             internal HandleLimitTester(HandleCollector collector)
             {
                 _collector = collector;
+                _pressure = new int[1_000];
```

Comment:
I know it is named `_pressure`, but can you also add a comment about the necessity of this.